### PR TITLE
Improve the test suite wrapper

### PR DESCRIPTION
**Pull request by matejak**
It now uses a whole, literal CPE for the purpose of CPE injection into the datastream.

The previous way that used part of the CPE stopped working for some reason.

### CODE REVIEW

**Comment by jan-cerny** [comment]
Looks fine.

> The previous way that used part of the CPE stopped working for some reason.

 In which situation? Do you have a reproducer?

File: tests/test_rule_in_container.sh [hunk 1]
```diff
@@ -185,7 +185,7 @@ script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)" || { echo "Couldn't d
 # [ <-- needed because of Argbash
 
 podman images | grep -q "$_arg_name" || die "Couldn't find the podman image '$_arg_name'"
-test_image_cpe_product=$(podman run --rm "$_arg_name" cat /etc/os-release | grep cpe | cut -d : -f 4)
+test_image_cpe_product=$(podman run --rm "$_arg_name" cat /etc/os-release | grep CPE_NAME | cut -f 2 -d = | tr -d "'\"")
```
Comment (cipherboy) [inline tests/test_rule_in_container.sh:188]: What about something like:

```
bash -c 'source /etc/os-release && echo $CPE_NAME'
```

`/etc/os-release` & friends were IIRC meant [to be source-able](https://www.freedesktop.org/software/systemd/man/os-release.html) and this is shorter. :-)

Comment (matejak) [reply tests/test_rule_in_container.sh]: So that's not so simple as source is a bash builtin, but something very similar works.

**Comment by matejak** [comment]
and another one that improves error reporting, esp. in connection with the Podman 2.2 issue.

**Comment by carlosmmatos** [comment]
I can confirm this addresses and fixes the issues faced with Python versions < 3.8. Also nice to have additional verbiage surrounding the podman version fiasco. If more people hit this issue, it may be prudent to add a link to some doc that goes more in-depth around this.

LGTM

**Pull request merged**